sources/packages/backend/apps/queue-consumers/test/helpers/mock-utils/cas-response.factory.ts

```diff
@@ -85,6 +86,17 @@ export function createFakeCASNotFoundSupplierResponse(): CASSupplierResponse {
   };
 }
 
+/**
+ * Creates a fake SendPendingInvoicesResponse.
```

Comment:
If the type is mentioned and available, please use the @link.
Please check the source code to see examples.